examples/getting_started/react/components/Todo/TodoContent.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+const Todo = ({ todo, editTodo, deleteTodo, toggleEdit }) => {
```

Comment:
Related to the previous comment: This filename here is `TodoContent` but the file is exporting a function called `Todo` but the parent component is also named `Todo` and that parent references this one as `Content`. While this is a small example, as an application grows this kind of approach to naming can quickly become confusing. This would be one of the arguments for totally avoiding using default exports but I'm not overly picky.

I guess this is not something you consciously did but you were probably just copy pasting stuff and it slipped past you. 😃 It would be a bit nicer if this were named `TodoContent`.

Reply:
This was actually one of those ones were I was debating the name for the component and what made the most sense. So at one stage it was Todo, then it became Content and then it became TodoContent and clearly I forgot to change it in each place. I think named exports will be slightly better, for the sake of maintenance.